addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -297,8 +292,8 @@ export class LinkPlugin extends Plugin {
 
         this.getExternalMetaData = memoize(fetchExternalMetaData);
         this.getInternalMetaData = memoize(fetchInternalMetaData);
-        this.getAttachmentMetadata = memoize((url) =>
-            fetchAttachmentMetaData(url, this.services.orm)
+        this.getAttachmentMetadata = memoize((url, fields = ["name", "mimetype"]) =>
```

Comment:
Hello! I'm not sure what you're doing here, but keep in mind that `memoize` only takes the first argument into account.
So a 2nd call to `getAttachmentMetadata` with the same `url` but different `fields` would return the same result of the first call (which might be missing a field's value, for example).